valve/addons/amxmodx/scripting/agmodx_llhl.sma

```diff
@@ -0,0 +1,430 @@
+    "fvox/powermove_on.wav",
+    "items/gunpickup2.wav",
+    "player/pl_dirt1.wav", "player/pl_dirt2.wav", "player/pl_dirt3.wav", "player/pl_dirt4.wav",
+    "player/pl_duct1.wav", "player/pl_duct2.wav", "player/pl_duct3.wav", "player/pl_duct4.wav",
+    "player/pl_fallpain3.wav",
+    "player/pl_grate1.wav", "player/pl_grate2.wav", "player/pl_grate3.wav", "player/pl_grate4.wav",
+    "player/pl_ladder1.wav", "player/pl_ladder2.wav", "player/pl_ladder3.wav", "player/pl_ladder4.wav",
+    "player/pl_metal1.wav", "player/pl_metal2.wav", "player/pl_metal3.wav", "player/pl_metal4.wav",
+    "player/pl_pain2.wav",
+    "player/pl_slosh1.wav", "player/pl_slosh2.wav", "player/pl_slosh3.wav", "player/pl_slosh4.wav",
+    "player/pl_step1.wav", "player/pl_step2.wav", "player/pl_step3.wav", "player/pl_step4.wav",
+    "player/pl_swim1.wav", "player/pl_swim2.wav", "player/pl_swim3.wav", "player/pl_swim4.wav",
+    "weapons/egon_off1.wav",
+    "weapons/egon_run3.wav",
+    "weapons/egon_windup2.wav"
+};
+
+public plugin_precache() {
+    if (!IsSelectedMode(MODE_TYPE_NAME)) {
+        StopPlugin();
+        return;
+    }
+}
+
+// We create the cvars before 'exec gamemodes/%s.cfg'
+public agmodx_pre_config() {
+    gCvarAllowVoteSetting = get_cvar_pointer("sv_ag_vote_setting");
+    // FPS Limiter
+    gCvarFpsCheckInterval = create_cvar("sv_ag_fpslimit_check_interval", "5.0");
+    gCvarFpsMaxDetections = create_cvar("sv_ag_fpslimit_max_detections", "5");
+    gCvarFpsMax = create_cvar("sv_ag_fpslimit_max_fps", "144");
+    gCvarFpsMarginError = create_cvar("sv_ag_fpslimiter_margin_error", "0.6");
+    // Unstuck command
+    gCvarUnstuckCooldown = create_cvar("sv_ag_unstuck_cooldown", "10.0");
+    gCvarUnstuckStartDistance = create_cvar("sv_ag_unstuck_start_distance", "32");
+    gCvarUnstuckMaxSearchAttempts = create_cvar("sv_ag_unstuck_max_attempts", "64");
+    // Sound file checker
+    gCvarCheckSoundFiles = create_cvar("sv_ag_check_soundfiles", "0");
+    // Destroyable Satchel
+    gCvarDestroyableSatchel =  create_cvar("sv_ag_destroyable_satchel", "0");
+    gCvarDestroyableSatchelHP = create_cvar("sv_ag_destroyable_satchel_hp", "1");
+    // Block name change (Only spectators) log in match
+    gCvarBlockNameChangeInMatch = create_cvar("sv_ag_block_namechange_inmatch", "1");
+
+    gCvarRespawnFix = get_cvar_pointer("mp_respawn_fix");
+
+    hook_cvar_change(gCvarFpsCheckInterval, "CvarAgFpsCheckIntervalHook");
+    hook_cvar_change(gCvarRespawnFix, "CvarAgRespawnFixHook");
+}
+
+public agmodx_post_config() {
+    if (get_pcvar_num(gCvarCheckSoundFiles)) {
+        // Sound file consistency
+        for (new i; i < sizeof gConsistencySoundFiles; i++) {
+            force_unmodified(force_exactfile, {0,0,0}, {0,0,0}, gConsistencySoundFiles[i]);
+        }
+    }
+}
+
+public plugin_init() {
+    register_plugin(PLUGIN, VERSION, AUTHOR);
+
+    register_dictionary("agmodxllhl.txt");
```

Comment:
I would change this to agmodx_lhll.txt to be more readable

Reply:
Sounds good